### ISSUE TITLE
jsonl and json confusion

### ISSUE THREAD

**Issue by olavski** [explosion/spaCy#3728]
`spacy convert` output now defaults to jsonl instead of json, so this example doesn't work anymore:

https://spacy.io/usage/training#spacy-train-cli
```python
git clone https://github.com/UniversalDependencies/UD_Spanish-AnCora
mkdir ancora-json
python -m spacy convert UD_Spanish-AnCora/es_ancora-ud-train.conllu ancora-json
python -m spacy convert UD_Spanish-AnCora/es_ancora-ud-dev.conllu ancora-json
mkdir models
python -m spacy train es models ancora-json/es_ancora-ud-train.json ancora-json/es_ancora-ud-dev.json
```
Also, it is not clear how to train a model with jsonl as `spacy train` seems to "expects data in spaCy’s JSON format".

------------------------
spaCy version    2.1.3
Platform         Linux-4.15.0-33-generic-x86_64-with-Ubuntu-18.04-bionic
Python version   3.6.5

**Comment by ines** [comment]
This was already fixed in #3583, sorry about that. In the meantime, you can set `-t json` on `spacy convert`. We're just building wheels for the upcoming release which will include the fixed default.

**Comment by olavski** [comment]
Thanks for the quick response!
